### PR TITLE
[FIX] web: implement filter orderedBy

### PR DESCRIPTION
**Pull request by kebeclibre**
related to: #28446

OPW 1906968

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
@aab-odoo 
ready for review

File: addons/web/static/src/js/views/basic/basic_model.js [hunk 1]
```diff
@@ -3332,7 +3332,7 @@ var BasicModel = AbstractModel.extend({
             model: params.modelName,
             offset: params.offset || (type === 'record' ? _.indexOf(res_ids, res_id) : 0),
             openGroupByDefault: params.openGroupByDefault,
-            orderedBy: params.orderedBy || [],
+            orderedBy: params.orderedBy || (params.context && params.context.orderedBy) || [],
```
Comment (kebeclibre) [inline addons/web/static/src/js/views/basic/basic_model.js]: @aab-odoo 
it didn't work directly when creating the view because of this

**Comment by kebeclibre** [comment]
@aab-odoo 
changes made

**Comment by aab-odoo** [comment]
robodoo r+ :+1:

**Pull request closed without merging**